test/fixtures/config-autoload/ts/array/postcss.config.cts

```diff
@@ -1,10 +1,10 @@
-import type { Config } from 'postcss-load-config';
+import type { Config as PostCSSConfig } from 'postcss-load-config';
+import { Configuration as WebpackConfig } from 'webpack';
 
-module.exports = function (api: Config) {
+module.exports = function (api: WebpackConfig): PostCSSConfig {
```

Comment:
Nope, it should be https://github.com/webpack-contrib/postcss-loader/blob/master/src/utils.js#L152, it is mix of types, so I think we just need to add this to documents

Reply:
Feel free to update types in docs and we can merge + small example, thank you